Backpack/Calendar/Classes/BPKCalendarCell.m

```diff
@@ -94,20 +94,20 @@ - (void)layoutSubviews {
 
     switch (self.rowType) {
     case RowTypeStart:
-        if (self.selectionType != SelectionTypeLeadingBorder) {
+        if (self.selectionType != SelectionTypeLeadingBorder && self.selectionType != SelectionTypeLeading) {
```

Comment:
This multiple condition seem to signal that there's something in design that we need to change, with a more polymorphic approach.

I'm thinking on maybe having 2 classes with a method each, or at least 2 methods here that would return a struct with `overflowLength, overflow and underflow` and manage these switches there, to isolate the functionality and avoid multiple if statements for every case.